ConwaysGameOfLife.Tests/GameControllerShould.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using Xunit;
+
+namespace ConwaysGameOfLife.Tests
+{
+    public class GameControllerShould
+    {
+        StubOutput _output;
+
+        StubInput _input;
+
+        public GameControllerShould()
+        {
+            _output = new StubOutput();
+            _input = new StubInput();
+        }
+        [Fact]
+        public void FormatUniverse_GivenUniverseOfAllDeadCells() 

```

Comment:
Are we testing the behaviour of the `GameController` or are we testing dependencies? e.g. we already have tests for `FormatUniverse`. The test itself is probably okay (if we are okay with having `DisplayUniverse` being a public function), just the name could be clearer that we're expecting a `Universe` being written to the `IOutput`